cf-runagent/cf-runagent.c

```diff
@@ -279,9 +279,7 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
             break;
 
         case 's':
-            strlcpy(SENDCLASSES, optarg, CF_MAXVARSIZE);
-
-            if (strlen(optarg) > CF_MAXVARSIZE)
+            if (strlcat(SENDCLASSES, optarg, CF_MAXVARSIZE) >= sizeof(SENDCLASSES))
```

Comment:
We prefer to use `sizeof(var)` where possible, instead of `CF_MAXVARSIZE` directly. I know there is a lot of legacy code doing the opposite, but we're trying to be consistent in the new code.